pkg/apis/core/v1/utils.go

```diff
@@ -363,3 +364,12 @@ func getActBackupStep(c *v1.Cluster, b *v1.Backup, bp *v1.BackupPoint, pNode *v1
 
 	return actBackup.GetStep(action), nil
 }
+
+func (h *handler) getBackupPoint(backups *v1.BackupList, name string) bool {
```

Comment:
update func name “existBackupPoint”

Reply:
I think maybe it would be better to change it to inUsed.